clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/BaseKeyValueOperation.java

```diff
@@ -25,8 +25,10 @@
 
   private final K key;
   private final V value;
+  private final long timeStamp;
+  private final byte isFirst;
```

Comment:
Removing this byte can be done by masking it in opcode as well and I have that as TODO already. So it does not make much difference